ui/pages/bridge/index.tsx

```diff
@@ -1,45 +1,57 @@
-import React, { useContext } from 'react';
+import React, { useContext, useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Switch, useHistory } from 'react-router-dom';
-import classnames from 'classnames';
 import { I18nContext } from '../../contexts/i18n';
-
 import { clearSwapsState } from '../../ducks/swaps/swaps';
 import {
   DEFAULT_ROUTE,
   SWAPS_MAINTENANCE_ROUTE,
   PREPARE_SWAP_ROUTE,
   CROSS_CHAIN_SWAP_ROUTE,
 } from '../../helpers/constants/routes';
-
 import { resetBackgroundSwapsState } from '../../store/actions';
-
 import FeatureToggledRoute from '../../helpers/higher-order-components/feature-toggled-route';
 import {
-  Box,
-  Icon,
+  ButtonIcon,
+  ButtonIconSize,
   IconName,
-  IconSize,
 } from '../../components/component-library';
-import {
-  JustifyContent,
-  IconColor,
-  Display,
-  BlockSize,
-} from '../../helpers/constants/design-system';
-import { getIsBridgeEnabled } from '../../selectors';
+import { getIsBridgeChain, getIsBridgeEnabled } from '../../selectors';
 import useBridging from '../../hooks/bridge/useBridging';
-import { PrepareBridgePage } from './prepare/prepare-bridge-page';
+import {
+  Content,
+  Footer,
+  Header,
+} from '../../components/multichain/pages/page';
+import { getProviderConfig } from '../../ducks/metamask/metamask';
+import { resetInputFields, setFromChain } from '../../ducks/bridge/actions';
+import { useSwapsFeatureFlags } from '../swaps/hooks/useSwapsFeatureFlags';
+import PrepareBridgePage from './prepare/prepare-bridge-page';
+import { BridgeCTAButton } from './prepare/bridge-cta-button';
 
 const CrossChainSwap = () => {
   const t = useContext(I18nContext);
 
+  useSwapsFeatureFlags();
```

Comment:
Why do we need the swaps feature flags?

Reply:
We need the swaps feature flags because they contain the Smart Transaction (STX) feature flags. So if we want Bridge to work with STX we need to load these. I can add a comment to clarify things in the code.